packages/product-download-page/index.tsx

```diff
@@ -179,7 +178,7 @@ export default function ProductDownloadsPage({
 
 interface ProductDownloadsPageProps {
   tutorialLink: Link
-  merchandisingSlot: React.ReactElement
+  merchandisingSlot?: React.ReactElement
```

Comment:
`merchandisingSlot` is a slot-type prop, I've added a check before rendering it for clarity, so it can now be safely set to optional.